### PR TITLE
Adding mfa screens

### LINKED ISSUE #9456: Add the MFA screens in login flow
This issue is part of the larger [MFA-implementation](https://github.com/ohcnetwork/care_fe/issues/6715). 

We decided to implement the TOTP with backup codes and eventually add support for other methods in future.

**Do not worry about the left half design, you can use the current sign in page design for the left half and just work on UI elements in the right half.**

Image

---

Image

- It is best to branch out from this [branch](https://github.com/ohcnetwork/care_fe/tree/form-field-v1) instead of develop.
- Currently the BE is not ready yet, wire it once it is ready.

> References:

- [CEP-MFA](https://github.com/ohcnetwork/docs/pull/81)
- [Full Figma Design](https://www.figma.com/design/Z93EYKSa1MdBmXndsMBJSQ/Care?node-id=1902-878&p=f&t=P5c4kFSL0PuAzk4Y-0) → For now only look at single role user flow.

### PR DESCRIPTION
**Pull request by AnveshNalimela**
## Proposed Changes

Fixes #9456  
Closes #9456  
Resolves #9456  

- Added a route for Autheticate
- Created Autheicate the account page
- More?

@ohcnetwork/care-fe-code-reviewers
Screenshot from 2024-12-23 19-57-06
Screenshot from 2024-12-23 19-57-15

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Introduced a new authentication route at `"/authenticate"` for user login.
	- Added a new `Authenticate` component that provides a user-friendly authentication interface with multiple login methods.

- **Accessibility Enhancements**
	- Improved accessibility features, including alt text for images and links.

Closes #9456

### CODE REVIEW

**Comment by AnveshNalimela** [comment]
Closes #9456

**Comment by Jacobjeevan** [comment]
@coderabbitai full review

File: src/components/Auth/Authenticate.tsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  const { urls, stateLogo, customLogo, customLogoAlt } = careConfig;
+  const customDescriptionHtml = __CUSTOM_DESCRIPTION_HTML__;
+  const logos = [stateLogo, customLogo].filter(
+    (logo) => logo?.light || logo?.dark,
+  );
+  const [recoveryModal, setRecoveryModal] = useState(false);
+
+  const accesWays: string[] = [
+    "OTP via SMS",
+    "OTP via Email",
```
Comment (yash-learner) [inline src/components/Auth/Authenticate.tsx]: Remove these two methods for now(V1) we are only implementing the TOTP with recovery codes as mentioned in the [issue](https://github.com/ohcnetwork/care_fe/issues/9456#:~:text=We%20decided%20to%20implement%20the%20TOTP%20with%20backup%20codes%20and%20eventually%20add%20support%20for%20other%20methods%20in%20future), we can add the remaining two when we support them.

**Comment by rithviknishad** [comment]
you can re-open this PR with develop as base branch @AnveshNalimela

**Comment by AnveshNalimela** [comment]
@rithviknishad  Couldnt able to reopen the PR

**Comment by rithviknishad** [comment]
new PR with develop as base branch, the form-field-v1 branch got merged and deleted, and since the base branch no longer exists, PR got auto closed.

**Pull request closed without merging**